### DISCUSSION TITLE
The next opcua pip update?

### DISCUSSION THREAD

**Discussion by tilinna** [FreeOpcUa/python-opcua#1278] [Q&A] [answered]
The current pip version listed in https://pypi.org/project/opcua/#history is pretty old. Are there plans to release an updated version soon?

**Comment by AndreasHeine** [comment, marked as answer]
The library is in maintenance mode. The plan is now to focus on the asyncio version of this library: opcua-asyncio which also has a sync-wrapper, with very few changes in API

**Reply by tilinna** [reply]
Fair enough, we'll take a look at the async version then. Thanks!

**Comment by oroulet** [comment]
I can update the pip version too. IT is not happening much anyway with python-opcua

**Reply by oroulet** [reply]
done
